agent/util-scripts/pbench-sysinfo-dump

```diff
@@ -96,20 +96,28 @@ function collect_sos {
 	fi
 }
 
+function collect_ara_data {
+	mkdir $dir/ara
+	ansible-playbook -vvv -i ${INVENTORY} --extra-vars '{"SYSINFO_DIR":"'$dir'"}' /opt/pbench-agent/ansible/ara/ara.yml
+}
+
 for item in ${sysinfo//,/ };do
-        if [[ "$item" == "kernel_config" ]]; then
+	if [[ "$item" == "kernel_config" ]]; then
```

Comment:
Looks like you have a tabs vs spaces problem.  This module should be indented by tabs, where tabs are equivalent to 8 spaces.